geomesa-spark/geomesa-spark-jts/src/test/scala/org/locationtech/geomesa/spark/jts/udf/GeometricConstructorFunctionsTest.scala

```diff
@@ -199,7 +200,7 @@ class GeometricConstructorFunctionsTest extends Specification with TestEnvironme
           |                    st_castToPoint(st_geomFromWKT('POINT(2 2)')))
         """.stripMargin
       )
-      val expected = WKTUtils.read("POLYGON((0.0 0.0, 2.0 0.0, 2.0 2.0, 0.0 2.0, 0.0 0.0))")
+      val expected = WKTUtils.read("POLYGON((0.0 0.0, 0.0 2.0, 2.0 2.0, 2.0 0.0, 0.0 0.0))")
```

Comment:
did axis order change here? that might be jarring to end users...

Reply:
@jnh5y what do you think? these axis order issues are always tricky. i'd say we should keep behavior consistent with longitude first, like we normally do with `Hints.putSystemDefault(Hints.FORCE_LONGITUDE_FIRST_AXIS_ORDER, true)`

Reply:
@jnh5y thoughts?

Reply:
This is a box:), so axis order and polygon winding order can be confused/conflated.

@tkunicki thoughts?

Also, can someone take it from the top for me?

Reply:
good point, maybe just the point order changed, not the axis order. maybe we could re-write the test with different lengths for x and y so we can tell? we'd have to run the original as well